policies/policy-adoption.md

```diff
@@ -1,19 +1,16 @@
-# Web of Things (WoT) Policy (Draft)
+# Web of Things (WoT) Policy
 ## Policy Adoption
+- First, a new issue should be created to motivate a new or change a Policy
+- New Policy should be discussed in the main call and there is a need of consensus. 
 - To propose a new policy, first create a PR against [wot/proposals/policies](https://github.com/w3c/wot/main/proposals/policies).
```

Comment:
the link is not valid, should be updated to: 
https://github.com/w3c/wot/tree/main/proposals/policies